pixel-point/nextjs-starter
# feat: add Register form

**Describe what changes this pull request brings**
- This pull request brings components Register Form, also shared components like Input, Heading.

**Describe any additional changes**
- added the next.config.js 
- installed @svgr/webpack to inline SVG (found the solution [here](https://github.com/vercel/next.js/discussions/14195))
- add the ready global styles, mixins from the repo [ghost-web-v2](https://github.com/pixel-point/ghost-web-v2)

**Steps to test**

1. Open the page
2. Confirm that everything looks and works as expected

**Screenshots**
![image](https://user-images.githubusercontent.com/48465000/103094726-2ab2ac00-4607-11eb-94c7-8979794f4248.png)


**Checklist**


**TODO**
- [ ] Add the left design part.

**References**
[Figma](https://www.figma.com/file/kYvLxXXcpPNhCt1SyUFnef/Ghost-Landing-Design?node-id=1563%3A3550)
